code/style.css

```diff
@@ -1,4 +1,312 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body{
+  background: #ffffff;
+  color: #515151;
+  font-family: quicksand;
+  margin: 0px;
 }
+
+h2 {
+  font-family: fredericka the great;
+  line-height: 32px;
+  font-size: 40px;
+  padding-right: 20px;
+  color: #515151;
+}
+
+p {
+  padding-right: 20px;
+  font-size: 20px;
+  color: #515151;
+}
+
+h3 {
+  font-size: 26px;
+  font-family: baskerville;
+}
+
+h4 {
+  font-size: 32px;
+  font-weight: 100;
+  margin-top: -16px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1400px;
+  margin: 0 auto;
```

Comment:
These two make your page look extra nice when it comes to the card-section. Just a heads up - it's also cropping your background for the main-article

Reply:
Ooo, and how I can keep that just for the card section? Thank you for your answer!

Reply:
One idea is to give the card section another class name to be able to have two lines of code that differs from the main container. Like:
```
.main-container, .container {
/*Styling applying to both*/
}

.container {
/*Styling applying for only the card section*/
}
```